ckanext/issues/model/__init__.py

```diff
@@ -391,13 +391,30 @@ def get_comment_count(cls, issue):
             filter(cls.issue_id == issue.id).count()
 
     @classmethod
-    def reported_comments(cls, session, organization_id):
-        return session.query(IssueComment, Issue) \
+    def reported_comments(cls, session, organization_id=None):
+        query = session.query(IssueComment, Issue) \
             .join(Issue) \
             .join(model.Package) \
             .filter(cls.visibility == u'hidden') \
             .filter(cls.abuse_status == AbuseStatus.unmoderated.value) \
-            .filter(model.Package.owner_org == organization_id)
+
+        if organization_id:
+            query = query.filter(model.Package.owner_org == organization_id)
+
+        return query
+
+    @classmethod
+    def unreported_comments(cls, session, organization_id=None):
+        query = session.query(IssueComment, Issue) \
+            .join(Issue) \
+            .join(model.Package) \
+            .filter(cls.visibility != u'hidden') \
+            .filter(cls.abuse_status == AbuseStatus.unmoderated.value) \
```

Comment:
This returns only the unmoderated ones. If that's what you want then I suggest you change the method name to reflect that - unreported_and_unmoderated_comments. I'm not clear why someone might want to only see unreported comments - I couldn't see how this got called.

BTW You might also clarify the reported_comments name to be reported_and_unmoderated_comments.